cellprofiler/modules/morph.py

```diff
@@ -603,6 +595,15 @@ def upgrade_settings(self, setting_values,
 
             variable_revision_number = 5
 
+        if variable_revision_number == 5:
+            # Removed "life" operation
+            logger.warn("""
```

Comment:
Python doesn't strip leading or trailing whitespace from `"""` messages, so the resulting log message looks like:

```
# previous load or start cp warning

            Morph's 'Life' option has been removed, this pipeline might
            not be compatible with the current version of CellProfiler.

```

Two approaches I know to prevent unnecessary newlines and whitespace in messages are:

1. Use `\` within `"""`s to escape line breaks and shove the text all the way left:
    ```python
         logger.warn("""\
    Morph's 'Life' option has been removed, this pipeline might \
    not be compatible with the current version of CellProfiler.""")
     ```

2. Use multiple single-quoted strings, which Python will automatically concatenate:

    ```python
     logger.warn(
         "Morph's 'Life' option has been removed, this pipeline might "
         "not be compatible with the current version of CellProfiler."
     )
    ```

Reply:
I could have sworn that I had found something online that stated that python multi-line strings didn't preserve that extra whitespace...perhaps that might have been for `argparse`. I'll fix that!